README.md

```diff
@@ -35,6 +35,7 @@ Creation IDs are used to digitally identify unique creations of many forms. They
 
 ## `0xBxxx_xxxx`
 * `0xB0B0_0000` [Unexpected Maker](https://github.com/unexpectedmaker)
+* `0xBA00_0000` [Espruino / BAngle.js](./creations/espruino.md)
```

Comment:
Is the capital `A` deliberate or just a typo?

```suggestion
* `0xBA00_0000` [Espruino / Bangle.js](./creations/espruino.md)
```

Reply:
I did it deliberately to imply how I can up with the creator id. I'm ok if you think it should be lowercase.

Reply:
It just looked like maybe a typo. You could add `BA for BAngle` or something like that on the rest of the line.

Reply:
Nah, I'll just switch it. Committed.